Makefile

```diff
@@ -35,6 +35,7 @@ style: clean
 fast_tests:
 	python -m pip install .[tests]
 	python -m pytest tests/test_gaudi_configuration.py tests/test_trainer_distributed.py tests/test_trainer.py tests/test_trainer_seq2seq.py
+	python -m pytest test_functional_text_generation_example.py
```

Comment:
I think it's a bit tricky to add this one to the fast tests now as it's going to be quite slow with the current infrastructure. Maybe we can comment this line and add a TODO comment to uncomment it when we have more servers dedicated to CI.

Reply:
updated